<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ir000781" name="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest" tests="66" failures="5" timestamp="19 Jun 2015 16:01:01 GMT" time="10.532" errors="26">
  <testcase name="untested_spec207_mustEnsureAllCallsOnItsSubscriptionTakePlaceFromTheSameThreadOrTakeCareOfSynchronization" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec207_mustEnsureAllCallsOnItsSubscriptionTakePlaceFromTheSameThreadOrTakeCareOfSynchronization(SubscriberWhiteboxVerification.java:246)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec207_mustEnsureAllCallsOnItsSubscriptionTakePlaceFromTheSameThreadOrTakeCareOfSynchronization(IdentityProcessorVerification.java:532)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec109_mayRejectCallsToSubscribeIfPublisherIsUnableOrUnwillingToServeThemRejectionMustTriggerOnErrorAfterOnSubscribe" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Skipping because no error state Publisher provided, and the test requires it. Please implement PublisherVerification#createFailedPublisher to run this test.">
      <![CDATA[org.testng.SkipException: Skipping because no error state Publisher provided, and the test requires it. Please implement PublisherVerification#createFailedPublisher to run this test.
	at org.reactivestreams.tck.PublisherVerification.potentiallyPendingTest(PublisherVerification.java:1147)
	at org.reactivestreams.tck.PublisherVerification.whenHasErrorPublisherTest(PublisherVerification.java:1136)
	at org.reactivestreams.tck.PublisherVerification.required_spec109_mayRejectCallsToSubscribeIfPublisherIsUnableOrUnwillingToServeThemRejectionMustTriggerOnErrorAfterOnSubscribe(PublisherVerification.java:506)
	at org.reactivestreams.tck.IdentityProcessorVerification.required_spec109_mayRejectCallsToSubscribeIfPublisherIsUnableOrUnwillingToServeThemRejectionMustTriggerOnErrorAfterOnSubscribe(IdentityProcessorVerification.java:285)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="untested_spec311_requestMaySynchronouslyCallOnCompleteOrOnError" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec311_requestMaySynchronouslyCallOnCompleteOrOnError(SubscriberWhiteboxVerification.java:448)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec311_requestMaySynchronouslyCallOnCompleteOrOnError(IdentityProcessorVerification.java:605)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_createPublisher3MustProduceAStreamOfExactly3Elements" time="0.003" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec202_shouldAsynchronouslyDispatch" time="0.000" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec202_shouldAsynchronouslyDispatch(SubscriberWhiteboxVerification.java:115)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec202_shouldAsynchronouslyDispatch(IdentityProcessorVerification.java:502)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_createPublisher1MustProduceAStreamOfExactly1Element" time="0.022" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="optional_spec111_maySupportMultiSubscribe" time="0.254" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@31a5c39e to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@5f2108b5 within 250 ms">
      <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@31a5c39e to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@5f2108b5 within 250 ms
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1180)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1121)
	at org.reactivestreams.tck.PublisherVerification.optional_spec111_maySupportMultiSubscribe(PublisherVerification.java:543)
	at org.reactivestreams.tck.IdentityProcessorVerification.optional_spec111_maySupportMultiSubscribe(IdentityProcessorVerification.java:300)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec203_mustNotCallMethodsOnSubscriptionOrPublisherInOnComplete" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_spec109_subscribeThrowNPEOnNullSubscriber" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingOneByOne" time="0.256" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@49993335 to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@5025a98f within 250 ms">
      <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@49993335 to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@5025a98f within 250 ms
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1180)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1121)
	at org.reactivestreams.tck.PublisherVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingOneByOne(PublisherVerification.java:557)
	at org.reactivestreams.tck.IdentityProcessorVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingOneByOne(IdentityProcessorVerification.java:305)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfront" time="0.257" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@cb644e to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@5cb9f472 within 250 ms">
      <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@cb644e to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@5cb9f472 within 250 ms
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1180)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1121)
	at org.reactivestreams.tck.PublisherVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfront(PublisherVerification.java:608)
	at org.reactivestreams.tck.IdentityProcessorVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfront(IdentityProcessorVerification.java:310)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec208_mustBePreparedToReceiveOnNextSignalsAfterHavingCalledSubscriptionCancel" time="0.253" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec213_failingOnSignalInvocation" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec213_failingOnSignalInvocation(SubscriberWhiteboxVerification.java:345)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec213_failingOnSignalInvocation(IdentityProcessorVerification.java:572)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec309_requestNegativeNumberMustSignalIllegalArgumentException" time="0.257" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_spec109_mustIssueOnSubscribeForNonNullSubscriber" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec315_cancelMustNotThrowExceptionAndMustSignalOnError" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec315_cancelMustNotThrowExceptionAndMustSignalOnError(SubscriberWhiteboxVerification.java:460)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec315_cancelMustNotThrowExceptionAndMustSignalOnError(IdentityProcessorVerification.java:615)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec312_cancelMustMakeThePublisherToEventuallyStopSignaling" time="0.507" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec108_possiblyCanceledSubscriptionShouldNotReceiveOnErrorOrOnCompleteSignals" time="0.000" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified by this TCK.">
      <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec108_possiblyCanceledSubscriptionShouldNotReceiveOnErrorOrOnCompleteSignals(PublisherVerification.java:442)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec108_possiblyCanceledSubscriptionShouldNotReceiveOnErrorOrOnCompleteSignals(IdentityProcessorVerification.java:270)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec203_mustNotCallMethodsOnSubscriptionOrPublisherInOnError" time="0.002" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec211_mustMakeSureThatAllCallsOnItsMethodsHappenBeforeTheProcessingOfTheRespectiveEvents" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec211_mustMakeSureThatAllCallsOnItsMethodsHappenBeforeTheProcessingOfTheRespectiveEvents(SubscriberWhiteboxVerification.java:333)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec211_mustMakeSureThatAllCallsOnItsMethodsHappenBeforeTheProcessingOfTheRespectiveEvents(IdentityProcessorVerification.java:562)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec306_afterSubscriptionIsCancelledRequestMustBeNops" time="0.257" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_spec102_maySignalLessThanRequestedAndTerminateSubscription" time="0.004" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_spec104_mustCallOnErrorOnAllItsSubscribersIfItEncountersANonRecoverableError" time="0.255" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <failure type="java.lang.AssertionError" message="Could not subscribe org.reactivestreams.tck.IdentityProcessorVerification$ManualSubscriberWithErrorCollection@43814d18 to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@610694f1 within 250 ms">
      <![CDATA[java.lang.AssertionError: Could not subscribe org.reactivestreams.tck.IdentityProcessorVerification$ManualSubscriberWithErrorCollection@43814d18 to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@610694f1 within 250 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flop(TestEnvironment.java:117)
	at org.reactivestreams.tck.TestEnvironment$Promise.expectCompletion(TestEnvironment.java:839)
	at org.reactivestreams.tck.TestEnvironment.subscribe(TestEnvironment.java:193)
	at org.reactivestreams.tck.TestEnvironment.subscribe(TestEnvironment.java:188)
	at org.reactivestreams.tck.IdentityProcessorVerification$3$1.<init>(IdentityProcessorVerification.java:395)
	at org.reactivestreams.tck.IdentityProcessorVerification$3.apply(IdentityProcessorVerification.java:390)
	at org.reactivestreams.tck.IdentityProcessorVerification$3.apply(IdentityProcessorVerification.java:387)
	at org.reactivestreams.tck.IdentityProcessorVerification.optionalMultipleSubscribersTest(IdentityProcessorVerification.java:694)
	at org.reactivestreams.tck.IdentityProcessorVerification.required_spec104_mustCallOnErrorOnAllItsSubscribersIfItEncountersANonRecoverableError(IdentityProcessorVerification.java:387)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </failure>
  </testcase>
  <testcase name="required_spec213_onNext_mustThrowNullPointerExceptionWhenParametersAreNull" time="0.002" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec316_requestMustNotThrowExceptionAndMustOnErrorTheSubscriber" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec316_requestMustNotThrowExceptionAndMustOnErrorTheSubscriber(SubscriberWhiteboxVerification.java:466)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec316_requestMustNotThrowExceptionAndMustOnErrorTheSubscriber(IdentityProcessorVerification.java:620)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec213_onError_mustThrowNullPointerExceptionWhenParametersAreNull" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="stochastic_spec103_mustSignalOnMethodsSequentially" time="0.199" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_spec213_onSubscribe_mustThrowNullPointerExceptionWhenParametersAreNull" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_spec303_mustNotAllowUnboundedRecursion" time="0.004" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="mustImmediatelyPassOnOnErrorEventsReceivedFromItsUpstreamToItsDownstream" time="0.099" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_spec308_requestMustRegisterGivenNumberElementsToBeProduced" time="0.257" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <failure type="java.lang.AssertionError" message="Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@43bd930a did not call `registerOnNext(5)` within 250 ms">
      <![CDATA[java.lang.AssertionError: Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@43bd930a did not call `registerOnNext(5)` within 250 ms
	at org.testng.Assert.fail(Assert.java:78)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:180)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.next(TestEnvironment.java:878)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectNext(SubscriberWhiteboxVerification.java:655)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectNext(SubscriberWhiteboxVerification.java:651)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$14.run(SubscriberWhiteboxVerification.java:428)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.subscriberTest(SubscriberWhiteboxVerification.java:486)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.required_spec308_requestMustRegisterGivenNumberElementsToBeProduced(SubscriberWhiteboxVerification.java:424)
	at org.reactivestreams.tck.IdentityProcessorVerification.required_spec308_requestMustRegisterGivenNumberElementsToBeProduced(IdentityProcessorVerification.java:595)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
Caused by: java.lang.AssertionError: Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@43bd930a did not call `registerOnNext(5)` within 250 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:177)
	... 32 more
]]>
    </failure>
  </testcase>
  <testcase name="required_spec210_mustBePreparedToReceiveAnOnErrorSignalWithoutPrecedingRequestCall" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec110_rejectASubscriptionRequestIfTheSameSubscriberSubscribesTwice" time="0.000" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified by this TCK.">
      <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec110_rejectASubscriptionRequestIfTheSameSubscriberSubscribesTwice(PublisherVerification.java:537)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec110_rejectASubscriptionRequestIfTheSameSubscriberSubscribesTwice(IdentityProcessorVerification.java:295)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="untested_spec314_cancelMayCauseThePublisherToShutdownIfNoOtherSubscriptionExists" time="0.000" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec314_cancelMayCauseThePublisherToShutdownIfNoOtherSubscriptionExists(SubscriberWhiteboxVerification.java:454)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec314_cancelMayCauseThePublisherToShutdownIfNoOtherSubscriptionExists(IdentityProcessorVerification.java:610)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="untested_spec204_mustConsiderTheSubscriptionAsCancelledInAfterRecievingOnCompleteOrOnError" time="0.000" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec204_mustConsiderTheSubscriptionAsCancelledInAfterRecievingOnCompleteOrOnError(SubscriberWhiteboxVerification.java:201)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec204_mustConsiderTheSubscriptionAsCancelledInAfterRecievingOnCompleteOrOnError(IdentityProcessorVerification.java:517)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec317_mustNotSignalOnErrorWhenPendingAboveLongMaxValue" time="0.255" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_spec309_requestZeroMustSignalIllegalArgumentException" time="0.255" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec107_mustNotEmitFurtherSignalsOnceOnErrorHasBeenSignalled" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified by this TCK.">
      <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec107_mustNotEmitFurtherSignalsOnceOnErrorHasBeenSignalled(PublisherVerification.java:436)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec107_mustNotEmitFurtherSignalsOnceOnErrorHasBeenSignalled(IdentityProcessorVerification.java:265)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec205_mustCallSubscriptionCancelIfItAlreadyHasAnSubscriptionAndReceivesAnotherOnSubscribeSignal" time="0.253" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec106_mustConsiderSubscriptionCancelledAfterOnErrorOrOnCompleteHasBeenCalled" time="0.000" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified by this TCK.">
      <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec106_mustConsiderSubscriptionCancelledAfterOnErrorOrOnCompleteHasBeenCalled(PublisherVerification.java:413)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec106_mustConsiderSubscriptionCancelledAfterOnErrorOrOnCompleteHasBeenCalled(IdentityProcessorVerification.java:255)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec209_mustBePreparedToReceiveAnOnCompleteSignalWithPrecedingRequestCall" time="0.256" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec310_requestMaySynchronouslyCallOnNextOnSubscriber" time="0.000" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec310_requestMaySynchronouslyCallOnNextOnSubscriber(SubscriberWhiteboxVerification.java:442)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec310_requestMaySynchronouslyCallOnNextOnSubscriber(IdentityProcessorVerification.java:600)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec105_mustSignalOnCompleteWhenFiniteStreamTerminates" time="0.264" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec212_mustNotCallOnSubscribeMoreThanOnceBasedOnObjectEquality_specViolation" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec212_mustNotCallOnSubscribeMoreThanOnceBasedOnObjectEquality_specViolation(SubscriberWhiteboxVerification.java:339)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec212_mustNotCallOnSubscribeMoreThanOnceBasedOnObjectEquality_specViolation(IdentityProcessorVerification.java:567)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_validate_boundedDepthOfOnNextAndRequestRecursion" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_exerciseWhiteboxHappyPath" time="0.257" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <failure type="java.lang.AssertionError" message="Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@3632be31 did not call `registerOnNext(5)` within 250 ms">
      <![CDATA[java.lang.AssertionError: Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@3632be31 did not call `registerOnNext(5)` within 250 ms
	at org.testng.Assert.fail(Assert.java:78)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:180)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.next(TestEnvironment.java:878)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectNext(SubscriberWhiteboxVerification.java:655)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectNext(SubscriberWhiteboxVerification.java:651)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$1.run(SubscriberWhiteboxVerification.java:78)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.subscriberTest(SubscriberWhiteboxVerification.java:486)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.required_exerciseWhiteboxHappyPath(SubscriberWhiteboxVerification.java:69)
	at org.reactivestreams.tck.IdentityProcessorVerification.required_exerciseWhiteboxHappyPath(IdentityProcessorVerification.java:492)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
Caused by: java.lang.AssertionError: Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@3632be31 did not call `registerOnNext(5)` within 250 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:177)
	... 32 more
]]>
    </failure>
  </testcase>
  <testcase name="required_spec317_mustSupportACumulativePendingElementCountUpToLongMaxValue" time="0.003" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfrontAndCompleteAsExpected" time="0.254" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@4b952a2d to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@69ea3742 within 250 ms">
      <![CDATA[org.testng.SkipException: Skipped because tested publisher does NOT implement this OPTIONAL requirement. Reason for skipping was: Could not subscribe org.reactivestreams.tck.TestEnvironment$ManualSubscriberWithSubscriptionSupport@4b952a2d to Publisher com.aol.simple.react.reactivestreams.FutureStreamProcessor@69ea3742 within 250 ms
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1180)
	at org.reactivestreams.tck.PublisherVerification.optionalActivePublisherTest(PublisherVerification.java:1121)
	at org.reactivestreams.tck.PublisherVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfrontAndCompleteAsExpected(PublisherVerification.java:641)
	at org.reactivestreams.tck.IdentityProcessorVerification.optional_spec111_multicast_mustProduceTheSameElementsInTheSameSequenceToAllOfItsSubscribersWhenRequestingManyUpfrontAndCompleteAsExpected(IdentityProcessorVerification.java:315)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec317_mustSupportAPendingElementCountUpToLongMaxValue" time="0.002" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_spec201_mustSignalDemandViaSubscriptionRequest" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="optional_spec104_mustSignalOnErrorWhenFails" time="0.003" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Skipping because no error state Publisher provided, and the test requires it. Please implement PublisherVerification#createFailedPublisher to run this test.">
      <![CDATA[org.testng.SkipException: Skipping because no error state Publisher provided, and the test requires it. Please implement PublisherVerification#createFailedPublisher to run this test.
	at org.reactivestreams.tck.PublisherVerification.potentiallyPendingTest(PublisherVerification.java:1147)
	at org.reactivestreams.tck.PublisherVerification.whenHasErrorPublisherTest(PublisherVerification.java:1136)
	at org.reactivestreams.tck.PublisherVerification.optional_spec104_mustSignalOnErrorWhenFails(PublisherVerification.java:346)
	at org.reactivestreams.tck.IdentityProcessorVerification.optional_spec104_mustSignalOnErrorWhenFails(IdentityProcessorVerification.java:240)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec307_afterSubscriptionIsCancelledAdditionalCancelationsMustBeNops" time="0.254" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec301_mustNotBeCalledOutsideSubscriberContext" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec301_mustNotBeCalledOutsideSubscriberContext(SubscriberWhiteboxVerification.java:418)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec301_mustNotBeCalledOutsideSubscriberContext(IdentityProcessorVerification.java:590)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="untested_spec304_requestShouldNotPerformHeavyComputations" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified by this TCK.">
      <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec304_requestShouldNotPerformHeavyComputations(PublisherVerification.java:791)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec304_requestShouldNotPerformHeavyComputations(IdentityProcessorVerification.java:330)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="untested_spec305_cancelMustNotSynchronouslyPerformHeavyCompuatation" time="0.001" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified by this TCK.">
      <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec305_cancelMustNotSynchronouslyPerformHeavyCompuatation(PublisherVerification.java:797)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec305_cancelMustNotSynchronouslyPerformHeavyCompuatation(IdentityProcessorVerification.java:335)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="optional_spec105_emptyStreamMustTerminateBySignallingOnComplete" time="0.262" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_spec107_mustNotEmitFurtherSignalsOnceOnCompleteHasBeenSignalled" time="0.257" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec206_mustCallSubscriptionCancelIfItIsNoLongerValid" time="0.000" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec206_mustCallSubscriptionCancelIfItIsNoLongerValid(SubscriberWhiteboxVerification.java:240)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec206_mustCallSubscriptionCancelIfItIsNoLongerValid(IdentityProcessorVerification.java:527)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec210_mustBePreparedToReceiveAnOnErrorSignalWithPrecedingRequestCall" time="0.002" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_spec209_mustBePreparedToReceiveAnOnCompleteSignalWithoutPrecedingRequestCall" time="0.253" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <failure type="java.lang.AssertionError" message="Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@80503 did not call `registerOnComplete()` within 250 ms">
      <![CDATA[java.lang.AssertionError: Subscriber com.aol.simple.react.reactivestreams.FutureStreamProcessor@80503 did not call `registerOnComplete()` within 250 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flop(TestEnvironment.java:117)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.expectCompletion(TestEnvironment.java:919)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectCompletion(SubscriberWhiteboxVerification.java:674)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectCompletion(SubscriberWhiteboxVerification.java:670)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$BlackboxProbe.expectCompletion(SubscriberWhiteboxVerification.java:666)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification$8.run(SubscriberWhiteboxVerification.java:290)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.subscriberTest(SubscriberWhiteboxVerification.java:486)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.required_spec209_mustBePreparedToReceiveAnOnCompleteSignalWithoutPrecedingRequestCall(SubscriberWhiteboxVerification.java:286)
	at org.reactivestreams.tck.IdentityProcessorVerification.required_spec209_mustBePreparedToReceiveAnOnCompleteSignalWithoutPrecedingRequestCall(IdentityProcessorVerification.java:547)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </failure>
  </testcase>
  <testcase name="required_spec313_cancelMustMakeThePublisherEventuallyDropAllReferencesToTheSubscriber" time="3.524" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_spec302_mustAllowSynchronousRequestCallsFromOnNextAndOnSubscribe" time="0.253" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_validate_maxElementsFromPublisher" time="0.000" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="untested_spec109_subscribeShouldNotThrowNonFatalThrowable" time="0.000" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <error type="org.testng.SkipException" message="Not verified by this TCK.">
      <![CDATA[org.testng.SkipException: Not verified by this TCK.
	at org.reactivestreams.tck.PublisherVerification.notVerified(PublisherVerification.java:1166)
	at org.reactivestreams.tck.PublisherVerification.untested_spec109_subscribeShouldNotThrowNonFatalThrowable(PublisherVerification.java:448)
	at org.reactivestreams.tck.IdentityProcessorVerification.untested_spec109_subscribeShouldNotThrowNonFatalThrowable(IdentityProcessorVerification.java:275)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec101_subscriptionRequestMustResultInTheCorrectNumberOfProducedElements" time="0.763" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest"/>
  <testcase name="required_mustRequestFromUpstreamForElementsThatHaveBeenRequestedLongAgo" time="0.254" classname="com.aol.simple.react.reactivestreams.TckProcessorVerificationTest">
    <failure type="java.lang.AssertionError" message="timeout while awaiting 5 within 250 ms">
      <![CDATA[java.lang.AssertionError: timeout while awaiting 5 within 250 ms
	at org.testng.Assert.fail(Assert.java:78)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:180)
	at org.reactivestreams.tck.TestEnvironment$Receptacle.next(TestEnvironment.java:878)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.nextElement(TestEnvironment.java:400)
	at org.reactivestreams.tck.TestEnvironment$ManualSubscriber.nextElement(TestEnvironment.java:396)
	at org.reactivestreams.tck.IdentityProcessorVerification$TestSetup.expectNextElement(IdentityProcessorVerification.java:724)
	at org.reactivestreams.tck.IdentityProcessorVerification$6$1.<init>(IdentityProcessorVerification.java:638)
	at org.reactivestreams.tck.IdentityProcessorVerification$6.apply(IdentityProcessorVerification.java:632)
	at org.reactivestreams.tck.IdentityProcessorVerification$6.apply(IdentityProcessorVerification.java:629)
	at org.reactivestreams.tck.IdentityProcessorVerification.optionalMultipleSubscribersTest(IdentityProcessorVerification.java:694)
	at org.reactivestreams.tck.IdentityProcessorVerification.required_mustRequestFromUpstreamForElementsThatHaveBeenRequestedLongAgo(IdentityProcessorVerification.java:629)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
Caused by: java.lang.AssertionError: timeout while awaiting 5 within 250 ms
	at org.testng.Assert.fail(Assert.java:89)
	at org.reactivestreams.tck.TestEnvironment.flopAndFail(TestEnvironment.java:177)
	... 34 more
]]>
    </failure>
  </testcase>
</testsuite>
